.github/workflows/test-examples.yml

```diff
@@ -44,3 +44,6 @@ jobs:
         run: pnpm build:examples
       - name: Run tests ✅
         run: pnpm test:examples
```

Comment:
I think you should change the trigger on this (to `pull_request`) as well, so that we don't end up with this snafu next time.

Reply:
fair, updated